docs/documentation/services/rest/svc-cargo/cancel-flight.api.mdx

```diff
@@ -0,0 +1,47 @@
+---
```

Comment:
This file is generated from the json files right?
If so, we don't want to add them to the git repo and have to put the *.api.mdx path in the .gitignore file

Reply:
correct, wasn't the intention. just figuring out how to populate the website if the json files are on hand, must have accidentally committed the generated mdxs